### ISSUE TITLE
[Windows 10] flycast (?) frames drops issue

### ISSUE THREAD

**Issue by Hari-82** [libretro/RetroArch#11460]
Hi, today i notice something strange, i will try to explain:

i was trying some games for dreamcast and sometimes i had  some noticeable frame drops, not in the numbers shown by retroarch (it's always stable at 60), resolution doesn't make any difference (1440x1080 or 640x480) so i do not think is a hardware problem.

this is not always happening, to replicate this issue i launch a game (virtua tennis) play a little, then close content, launch another game (virtua tennis 2) and the fps drops happen, or sometimes it happens with the 3rd launch...

im using vulkan as back-end, i tried to change hardware shared context on/off, no difference.

im on 1.9.0 in  windows 10 64bit,  i5 6400,  8 GB of ram and GTX 960

i do not think is something related to hardware because when i first launch the game @1440x1080 everything runs very smooth at 60fps.

**Edit**
after some more testing i can replicate the issue 100% of the time

1) launch the game ->load state to gameplay [full speed, no problem]
2) close content
3) launch the game ->load state to gameplay [full speed, no problem]
4) close content
5) launch the game ->load state to gameplay [random frames drops for few seconds (feels like 20) while the fps on screen says 60]

to be able to play again full speed i have to quit retroarch and re open it.

any ideas?

**Comment by Ryunam** [comment]
Can you confirm if by chance you are using the Vulkan video driver and you also have MSI Afterburner / RTSS installed? 

If that is the case, you should update to the latest beta of RTSS. Recent changes to that application fixed an underlying issue with the Vulkan bootstrap layer files, which used to cause a very similar issue to the one that you just described. See here: https://github.com/libretro/RetroArch/issues/9668#issuecomment-642108698

**Comment by Hari-82** [comment]
yes, i use vulkan as video driver and i have both, afterburner and RTSS, running in the background.
later tonight i will try to update RTSS and give feedback!

thank you so much for the feedback!

**Comment by Hari-82** [comment]
> Can you confirm if by chance you are using the Vulkan video driver and you also have MSI Afterburner / RTSS installed?
> 
> If that is the case, you should update to the latest beta of RTSS. Recent changes to that application fixed an underlying issue with the Vulkan bootstrap layer files, which used to cause a very similar issue to the one that you just described. See here: [#9668 (comment)](https://github.com/libretro/RetroArch/issues/9668#issuecomment-642108698)

this solved the issue, thanks a lot!

also, before installing the RTSS beta mentioned i tried to simply disable both afterburner & RTSS and still i had the issue, but that went away after insalling the 7.3.0 beta 5.
then after reading the whole thread u suggested  i found that the same was happening to other people...

nonetheless, thanks again for the solution!

**Comment by Ryunam** [comment]
Definitely happy to hear that this fixed it! 

If you look at that thread, I spent so long trying to investigate this and understand why the stutter and frame drop was triggering even with RTSS completely disabled. Turns out that the sole presence of the bootstrap layer file, even when RTSS was not open or active at all, caused cores using Vulkan to stutter upon consecutive content loads. The RTSS developer was kind enough to look into this matter and apply a fix in the next beta builds.

I think this issue can be closed. I see that you have reopened it, but if there are no remaining problems maybe we can consider this addressed for now. :)
